src/main/webapp/resources/js/pages/analysis/components/GalaxyJobInfoTab.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { PassTabs } from "./PassTabs";
+import { GalaxyJobInfo } from "./GalaxyJobInfo";
+import { Col, Typography } from "antd";
+import { getI18N } from "../../../utilities/i18n-utilties";
+
+const { Title } = Typography;
+
```

Comment:
Make sure you have comments on the top of files says how or why they are used.

Reply:
Added in 39c89d161e41dc4241d4d58b6333be75bf9d2a92